### DISCUSSION TITLE
Sharing - Select all users

### DISCUSSION THREAD

**Discussion by igg55555** [LycheeOrg/Lychee#2451] [Q&A] [answered]
Please create an option to share with all users.  I am wanting all of my albums to be private, but available to all users.  I need to create 100+ users and will be adding more occasionally.  99 percent of these users will all be given them same permissions.

Having to go through every album and sub-album to share to a new user every time one is added would be incredibly cumbersome.

**Comment by ildyria** [comment, marked as answer]
@igg55555 https://github.com/LycheeOrg/Lychee/pull/2461

I think this pull request should make you happy.

Different approach, same result.

**Reply by ildyria** [reply]
This is now live.
